vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoUtility.java

```diff
@@ -0,0 +1,1426 @@
+         */
+        public static final class Top {
+
+            public static final String NONE = "pt-0";
+            public static final String XSMALL = "pt-xs";
+            public static final String SMALL = "pt-s";
+            public static final String MEDIUM = "pt-m";
+            public static final String LARGE = "pt-l";
+            public static final String XLARGE = "pt-xl";
+
+            private Top() {
+            }
+        }
+
+        /**
+         * Classes for defining both the vertical and the horizontal paddings of
+         * an element.
+         */
+        public static final class Vertical {
+
+            public static final String NONE = "py-0";
+            public static final String XSMALL = "py-xs";
+            public static final String SMALL = "py-s";
+            public static final String MEDIUM = "py-m";
+            public static final String LARGE = "py-l";
+            public static final String XLARGE = "py-xl";
+
+            private Vertical() {
+            }
+        }
+
+    }
+
+    /**
+     * Classes for setting the position of an element.
+     */
+    public static final class Position {
+
+        public static final String ABSOLUTE = "absolute";
+        public static final String FIXED = "fixed";
+        public static final String RELATIVE = "relative";
+        public static final String STATIC = "static";
+        public static final String STICKY = "sticky";
+
+        private Position() {
+        }
+
+    }
+
+    /**
+     * Classes for setting an element’s text alignment.
+     */
+    public static final class TextAlignment {
+
+        public static final String LEFT = "text-left";
+        public static final String CENTER = "text-center";
+        public static final String RIGHT = "text-right";
+        public static final String JUSTIFY = "text-justify";
+
+        private TextAlignment() {
+        }
+
+    }
+
+    /**
+     * Classes for setting an element’s text color.
+     */
+    public static final class TextColor {
+
+        public static final String HEADER = "text-header";
+        public static final String BODY = "text-body";
+        public static final String SECONDARY = "text-secondary";
+        public static final String TERTIARY = "text-tertiary";
+        public static final String DISABLED = "text-disabled";
+
+        public static final String PRIMARY = "text-primary";
+        public static final String PRIMARY_CONTRAST = "text-primary-contrast";
+
+        public static final String ERROR = "text-error";
+        public static final String ERROR_CONTRAST = "text-error-contrast";
+
+        public static final String SUCCESS = "text-success";
+        public static final String SUCCESS_CONTRAST = "text-success-contrast";
+
+        private TextColor() {
+        }
+
+    }
+
+    /**
+     * Classes for setting the text overflow.
+     */
+    public static final class TextOverflow {
+
+        public static final String CLIP = "overflow-cip";
```

Comment:
Typo: `cip` -> `clip`

```suggestion
        public static final String CLIP = "overflow-clip";
```

Reply:
Done.